setup.py

```diff
@@ -96,4 +96,5 @@
             'sample=sample:main',
         ],
     },
+    zip_safe=True,
```

Comment:
I have explained it on Slack, but whatever: Concerning the `zip_safe` thing, I recall that we have agreed to keep it in the PR on Friday. By the way, it actually was relevant for the PR (as the `PyRoma` bear was complaining about it).